scripts/easy_motion.sh

```diff
@@ -120,9 +140,20 @@ easy_motion_cleanup() {
 }
 
 main() {
+    local session_id window_id pane_id
+    session_id="$2"
+    window_id="$3"
+    pane_id="$4"
+
+    if [[ "${session_id}" =~ \$(.*) ]]; then
+        session_id="${BASH_REMATCH[1]}"
+    fi
+
+    set -- "$1" "${session_id}" "${@:3}"
+
     read_options && \
-    easy_motion_setup && \
-    easy_motion "$@"
+    easy_motion_setup "${session_id}" "${window_id}" "${pane_id}" && \
+    easy_motion "$@" && \
```

Comment:
```suggestion
    easy_motion "$@"
```
The cleanup step after `easy_motion` should always be run.

Reply:
Fixed.